.azure/pipelines/blazor-daily-tests.yml

```diff
@@ -0,0 +1,59 @@
+# Uses Scheduled Triggers, which aren't supported in YAML yet.
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=vsts&tabs=yaml#scheduled
+
+# Daily Tests for Blazor
+# These use Sauce Labs resources, hence they run daily rather than per-commit.
+
+# We just need one Windows machine because all it does is trigger SauceLabs.
+variables:
+  SAUCE_CONNECT_DOWNLOAD_ON_INSTALL: true
+  E2ETESTS_SauceTest: true
+  E2ETESTS_Sauce__TunnelIdentifier: 'blazor-e2e-sc-proxy-tunnel'
```

Comment:
Out of curiosity, why the double underscore?

Reply:
That's how config works in ASP.NET Core, it's the separator for config keys